src/components/select.tsx

```diff
@@ -0,0 +1,26 @@
+import { ChangeEvent } from 'react';
+
+function Select({
+  options,
+  onChange,
+  defaultValue,
+}: {
+  options: {
+    key: string;
+    value: string;
+  }[];
+  onChange: ({ target }: ChangeEvent<HTMLSelectElement>) => void;
+  defaultValue: string;
+}) {
+  return (
+    <select defaultValue={defaultValue} onChange={onChange} className="text-backgrounds-main">
+      {options.map((option) => (
+        <option key={option.key} value={option.value}>
```

Comment:
```suggestion
        <option key={option.key} value={option.key}>
```

klucz ma być taką wartością taka która faktyczni będzie służyła do wysyłania, przechowywania
`value` ma służyć tylko do wyświetlania wartości. tak jak pisałem, że może `{value: "WARTOŚĆ", label: "Labelka}` lepiej by pasowało, żeby nie było mylące